app/models/contact_email.rb

```diff
@@ -0,0 +1,5 @@
+class ContactEmail < ContactPoint
+  def self.label_collection
```

Comment:
so why where the roles in user to be named role_collection and now there is a unspeaking for_select necessary? What if there would be more than one in the same model?